packages/scripts/package.json

```
@@ -79,6 +79,7 @@
 		"react-refresh": "^0.14.0",
 		"read-pkg-up": "^7.0.1",
 		"resolve-bin": "^0.4.0",
+		"rtlcss-webpack-plugin": "4.0.7",
```

Let's ensure we don't force people to pin this particular version:
```suggestion
		"rtlcss-webpack-plugin": "^4.0.7",
```